tools/test-js-with-node

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-set -e
```

Comment:
It makes me a little nervous to turn off `set -e`, but I think it's worth it for the simplification here.  This change looks good to me.